autogpt_platform/backend/backend/data/model.py

```diff
@@ -112,6 +112,7 @@ def SchemaField(
     placeholder: Optional[str] = None,
     advanced: Optional[bool] = None,
     secret: bool = False,
+    is_multi_select: bool = False,
```

Comment:
This seems unnecessary to me. `list[SomeEnum]` is always going to be multi-select.